x-pack/elastic-agent/spec/apm-server.yml

```diff
@@ -32,4 +32,5 @@ rules:
         - output
         - fleet
   - inject_headers: {}
-when: length(${inputs}) > 0 and hasKey(${output}, 'elasticsearch')
+when: length(${inputs}) > 0 and hasKey(${output}, 'elasticsearch', 'redis',
+  'kafka', 'logstash')
```

Comment:
My information was that `redis` will not be supported for now?

Reply:
just copy and paste from the other Beats. I would like to keep all libbeat based process types in sync. Removing redis should be done separately.